### DISCUSSION TITLE
blobfuse2-2.3.0 not available for RHEL 7.9

### DISCUSSION THREAD

**Discussion by rajsshah86** [Azure/azure-storage-fuse#1419] [Q&A] [answered]
Hi 
I see that blobfuse2-2.3.0 is available for rhel 7.5 and 7.8 but not for the last version 7.9 . Is there a specific reason for it ? Do I have only the option to build it ?
We are using  
```
rpm -Uvh https://packages.microsoft.com/config/rhel/7/packages-microsoft-prod.rpm
```
to install the repo 

so in short question is why package is not published under **https://packages.microsoft.com/rhel/7/prod/** because that is the repo config you get when you install above rpm based on [MS suggestion ](https://learn.microsoft.com/en-us/azure/storage/blobs/blobfuse2-how-to-deploy?tabs=RHEL#configure-the-microsoft-package-repository)

**Comment by souravgupta-msft** [comment, marked as answer]
Hi @rajsshah86. We release blobfuse2 on several RHEL version like 7.0, 7.5, 7.8, 8.0, etc. For RHEL 7.9 you can download the rpm file from [RHEL 7.8](https://packages.microsoft.com/rhel/7.8/prod/Packages/b/blobfuse2-2.3.0-1.x86_64.rpm) and install it. Regarding the documentation, we will update it.
